static/js/settings_users.js

```diff
@@ -524,7 +525,19 @@ export function confirm_reactivation(user_id, handle_confirm, loading_spinner) {
     const opts = {
         username: user.full_name,
     };
-    const html_body = render_settings_reactivation_user_modal(opts);
+
+    const bot = bot_data.get(user_id);
```

Comment:
Would checking `user.is_bot` work instead? I think that'd be cleaner than doing a separate lookup here.